setup.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (C) 2019, RS
+# Full license can be found in License.md
+# -----------------------------------------------------------------------------
+
+import sys
+import os
+from setuptools import setup, find_packages
+
+
+here = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(here, 'description.txt'), encoding='utf-8') as f:
+    long_description = f.read()
+version_filename = os.path.join('pysatModelUtils', 'version.txt')
+with open(os.path.join(here, version_filename)) as version_file:
+    version = version_file.read().strip()
+
+# change setup.py for readthedocs - commented for now
+# on_rtd = os.environ.get('READTHEDOCS') == 'True'
+if sys.version_info.major == 2:
+    install_requires = ['xarray<0.12', 'pandas>=0.23, <0.25',
+                        'numpy>=1.12, <1.17', 'scipy<1.3', 'pysat']
+else:
+    install_requires = ['xarray', 'pandas>=0.23, <0.25', 'numpy>=1.12',
+                        'pysat']
```

Comment:
system requirements copied from pysat for now.  Will refine before 0.1.0 release